### ISSUE TITLE
No ability to run svc commands

### ISSUE THREAD

**Issue by kriss9** [jehiah/go-daemontools#2]
Currently, the documentation references the ability to perform `svc` commands. However, no functions suggest this is a possible way to interact with daemontools environment. Hence, this item is to add svc support to `go-daemontools`.

**Comment by jehiah** [comment]
Good Point!  Do you have any interested in adding a contribution for that? I haven't yet really needed that functionality in Go yet.
